common/core/gpudevice.cpp

```diff
@@ -57,13 +72,19 @@ bool GpuDevice::Initialize() {
     display_manager_->RemoveUnreservedPlanes();
   }
 
-  lock_fd_ = open("/vendor/hwc.lock", O_RDONLY);
+  // Ignore all updates by default
+  ResetAllDisplayCommit(false);
+  lock_fd_ = open(HWC_LOCK_FILE, O_RDONLY);
   if (-1 != lock_fd_) {
     if (!InitWorker()) {
       ETRACE("Failed to initalize thread for GpuDevice. %s", PRINTERROR());
     }
   } else {
-    ITRACE("Failed to open " LOCK_DIR_PREFIX "/hwc.lock file!");
+    ITRACE("Failed to open %s", HWC_LOCK_FILE);
```

Comment:
To avoid confusion, it is better to use a neutral word instead of “Failed to” if it is not an error.

Reply:
The patch in selinux is committed.
https://github.com/projectceladon/device-intel-sepolicy/pull/7
We will hold to merge this one until selinux one is merged

Reply:
Updated for log level